Makefile

```diff
@@ -20,10 +20,10 @@ test_bats: warp $(BATS_FILES) tests/cli/*.bats
 test_yul: warp
```

Comment:
I think you meant to name the commit "remove pytest-parallel" =)

Reply:
oops yea I'll amend it, just investigating seems that develop has failing CI so when I rebased my CI also started failing.
I'll push the amended the commit message with my changes to fix CI

Reply:
Hmm, I think that it happens when web3 updated. Can you try `poetry add web3==5.12.2`?

Reply:
I guess, we need to set a lower bound, like web3 = ">=5.12"

Reply:
done, thanks for the tip